src/controllers/commands/CommandController.cpp

```diff
@@ -452,6 +453,48 @@ void CommandController::initialize(Settings &, Paths &paths)
 
             return "";
         });
+
+    this->registerCommand("/marker", [](const QStringList &words,
+                                        auto channel) {
+        // Avoid Helix calls without Client ID and/or OAuth Token
+        if (getApp()->accounts->twitch.getCurrent()->isAnon())
+        {
+            channel->addMessage(makeSystemMessage(
+                "You need to be logged in to create stream markers!"));
+        }
+
+        auto *twitchChannel = dynamic_cast<TwitchChannel *>(channel.get());
```

Comment:
Please check the return value of `dynamic_cast` to ensure this doesn't crash. Even if non-`TwitchChannel`s don't currently use `CommandController`, it's still a sane thing to do

Reply:
I am a little confused because every command is using this solution everywhere.